packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 50.1.0 - October 14, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Update LWC preview command to launch a native app or mobile browser ([PR #2585](https://github.com/forcedotcom/salesforcedx-vscode/pull/2585))
```

Comment:
should we show real name of command? SFDX: Preview Component Locally ?? Is this the one? I vote for real name